cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -245,7 +246,8 @@ func init() {
 	Cmd.Flags().StringVar(&pkg.ImageCMDOptions.ImageName, "image-name", "", "Name of the resultant OVA image")
 	Cmd.Flags().StringVar(&pkg.ImageCMDOptions.ImageURL, "image-url", "", "URL or absolute local file path to the <QCOW2>.gz image")
 	Cmd.Flags().StringVar(&pkg.ImageCMDOptions.ImageDist, "image-dist", "", "Image Distribution(supported: rhel, centos, coreos)")
-	Cmd.Flags().Uint64Var(&pkg.ImageCMDOptions.ImageSize, "image-size", 120, "Size (in GB) of the resultant OVA image")
+	Cmd.Flags().Uint64Var(&pkg.ImageCMDOptions.ImageSize, "image-size", 3, "Size (in GB) of the resultant OVA image")
+	Cmd.Flags().Int64Var(&pkg.ImageCMDOptions.TargetDiskSize, "target-disk-size", 120, "Size (in GB) of the target disk volume where OVA will be copied")
```

Comment:
nitpick: we can use Uint64Var to maintain code uniformity

Reply:
@bkhadars the OVA template expects the values to be in Int64. If we make Uint64 here, then we will need to add a conversion. I can do that if you prefer it. Let me know

Reply:
ok